tests/foreman/smoke/test_api_smoke.py

```diff
@@ -167,6 +167,19 @@
         u'/katello/api/organizations/:organization_id/content_views',
         u'/katello/api/organizations/:organization_id/content_views',
     ),
+    u'containers': (
+        u'/docker/api/v2/containers',
```

Comment:
This is how API returns, each entry means a HTTP method. If an URL accepts GET and POST then it will be returned twice.

I will sort the URLS to be like others.